File: store-admin/src/main/java/com/delivery/storeadmin/domain/userorder/controller/model/UserOrderDetailResponse.java
package com.delivery.storeadmin.domain.userorder.controller.model;

import com.delivery.storeadmin.domain.storemenu.controller.model.StoreMenuResponse;
import java.util.List;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class UserOrderDetailResponse {

    private UserOrderResponse userOrderResponse;

    private List<StoreMenuResponse> storeMenuResponseList;

}

File: api/src/main/kotlin/com/delivery/api/domain/userorder/controller/model/UserOrderDetailResponse.kt
package com.delivery.api.domain.userorder.controller.model

import com.delivery.api.domain.store.controller.model.StoreResponse
import com.delivery.api.domain.storemenu.controller.model.StoreMenuResponse

data class UserOrderDetailResponse (
    var userOrderResponse: UserOrderResponse? = null,
    var storeResponse: StoreResponse? = null,
    var storeMenuResponseList: List<StoreMenuResponse>? = null,
)